quixstreams/sinks/community/postgresql.py

```diff
@@ -178,15 +183,15 @@ def _init_table(self):
             )
             """
         ).format(
-            table=sql.Identifier(self.table_name),
+            table=sql.Identifier(table_name),
```

Comment:
Please add here a support for schema. Expose this in contrstructor and also connect that to environment variable in template.

Example:
```python
sql.Identifier(self.schema_name, self.table_name)
```